daemon/mgr/image_utils.go

```diff
@@ -25,9 +27,16 @@ func containerdImageToOciImage(ctx context.Context, img containerd.Image) (ocisp
 		return ocispec.Image{}, err
 	}
 
+	// NOTE(fuweid): There is config content with legacy media type in
+	// content storage. In order to compatible with existing image,
+	// we should support it.
+	//
+	// more information is here: https://github.com/containerd/containerd/pull/2814
 	switch cfg.MediaType {
-	case ocispec.MediaTypeImageConfig, images.MediaTypeDockerSchema2Config:
-		data, err := content.ReadBlob(ctx, img.ContentStore(), cfg.Digest)
+	case ocispec.MediaTypeImageConfig, images.MediaTypeDockerSchema2Config,
```

Comment:
ping @HusterWan and @Ace-Tang , there is used to support existing downloaded images, for example, redis:2 with the `application/octet-stream` media type. if we don't support it here, we will miss the image.

Reply:
cc @rudyfly

Reply:
Agree that